update tikv-server-config file with bottommost-level-compression (#4480)
pingcap/docs: tikv-configuration-file.md

+ Optional values: ["no", "no", "lz4", "lz4", "lz4", "zstd", "zstd"]
+ Default value: `No` for the first two levels, and `lz4` for the next five levels

### `bottommost-level-compression`

> + Sets the compression algorithm of the bottom most layer. This configuration item overrides the setting in the `compression-per-level`.

Reviewer: ```suggestion
+ Sets the compression algorithm of the bottommost layer. This configuration item overrides the `compression-per-level` setting.
```

> + Ever since data is written to LSM-tree, TiKV does not directly adopt the last compression algorithm specified in the `compression-per-level` array for the bottommost layer. `bottommost-level-compression` enables the bottommost layer to use the compression algorithm of the best compression effect from the beginning.

Reviewer 2: TiKV -> RocksDB

> + The bottom most layer does not directly adopt the last compression algorithm in the `compression-per-level` array from when the data is written to LSM tree; therefore, `bottommost-level-compression` enables the bottom most layer to use the compression algorithm with the best compression effect from the beginning.

Reviewer: ```suggestion
+ Ever since data is written to LSM-tree, TiKV does not directly adopt the last compression algorithm specified in the `compression-per-level` array for the bottommost layer. `bottommost-level-compression` enables the bottommost layer to use the compression algorithm of the best compression effect from the beginning.
```

> + If you don't want to set the compression algorithm of the bottom most layer, set the value of this configuration item to `disable`.

Reviewer: ```suggestion
+ If you do not want to set the compression algorithm for the bottommost layer, set the value of this configuration item to `disable`.
```
